access-common/src/main/java/edu/unc/lib/dl/ui/util/StringFormatUtil.java

```diff
@@ -17,84 +17,155 @@
 
 import java.io.UnsupportedEncodingException;
 import java.net.URLEncoder;
-import java.util.regex.Matcher;
-import java.util.regex.Pattern;
 import java.nio.charset.StandardCharsets;
 
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
 /**
- * String manipulation utility methods related to Solr data.
+ * String manipulation utility methods for formatting and rendering string data
+ * to web clients.
+ *
  * @author bbpennel
  *
  */
 public class StringFormatUtil {
+    private static final Logger LOG = LoggerFactory.getLogger(StringFormatUtil.class);
 
     private static String[] filesizeSuffixes = {"B", "KB", "MB", "GB", "TB", "PB", "EB", "ZB", "YB"};
```

Comment:
pretty sure we'll never need to handle ZB or YB amounts of data in the lifetime of the CDR :-)

Reply:
Better safe than sorry 👍